cli/cluster/config_test.go

```diff
@@ -313,9 +313,9 @@ foo:
   bar: 1
   zoo: 1
 zoo:
-- "1"
-- "2"
-- "3"
+  - "1"
```

Comment:
Why are the changes required? As I could see the Tarantool prefers no indentation here in examples:
https://github.com/tarantool/tarantool/blob/master/doc/examples/config/replicaset_manual_failover.yaml#L23

Reply:
Because of the way how yaml v3 encodes values: https://go.dev/play/p/LaNfVBcRP2J
And here is no way to configure this behavior or I can't see it in the public API: https://pkg.go.dev/gopkg.in/yaml.v3#section-readme

Reply:
Does Tarantool read updated configs fine?

Reply:
Yes, here are no problems.

Reply:
We use the same format with indents in some integration tests: https://github.com/tarantool/tt/blob/master/test/integration/running/small_cluster_app/config.yaml

Reply:
https://github.com/tarantool/tarantool/blob/d7b72654799272e541882bea3e16ea79b1b7a36d/doc/examples/config/etcd/local.yaml#L9